esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -2016,18 +2013,18 @@ void WsWuInfo::getWorkunitThorSlaveLog(IGroup *nodeGroup, const char *ipAddress,
     dir.clear();
```

Comment:
Not new: curious why it opens an IDirectoryIterator here, then gets 1st..
why not just open file directly (createIFile) ?

Reply:
Will open file directly. Not sure why it does not do that before.